apps/src/javalab/javalabRedux.js

```diff
@@ -1,9 +1,11 @@
 const APPEND_CONSOLE_LOG = 'javalab/APPEND_CONSOLE_LOG';
 const SET_EDITOR_TEXT = 'javalab/SET_EDITOR_TEXT';
+const SET_FILE_NAME = 'javalab/SET_FILE_NAME';
 
 const initialState = {
   consoleLogs: [],
-  editorText: ''
+  editorText: '',
+  fileName: 'Untitled File'
```

Comment:
Ignore if this is too specific for this point in the work:
Would a default like "Main.java" or "MyClass.java" make sense? Thinking it might prevent some compiler errors and would let us insert some starter code like this:
`class MyClass {}`
Might also help reinforce the concept that a file is a Java class.

Reply:
good point, I think MyClass.java makes sense.